ui/raidboss/timeline.js

```diff
@@ -277,11 +285,28 @@ class Timeline {
       }
     }
 
-    for (let i = 0; i < this.events.length; ++i) {
-      let e = this.events[i];
+    // Validate that all timeline triggers match something.
+    if (triggers) {
+      for (const trigger of triggers) {
+        let found = false;
+        for (const event of this.events) {
+          let m = event.name.match(trigger.regex);
+          if (m) {
```

Comment:
nit: This feels odd since we don't actually use `m`.

Reply:
Done, thanks!

Reply:
should've been `regExp.test` too 👀 , it's a much faster boolean check